sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sampling_algorithm.py

```diff
@@ -39,16 +45,24 @@ def _from_rest_object(cls, obj: RestSamplingAlgorithm) -> "SamplingAlgorithm":
 class RandomSamplingAlgorithm(SamplingAlgorithm):
     """Random Sampling Algorithm.
 
-    :ivar type: Specifies the type of sampling algorithm. Set automatically to "random" for this class.
-    :vartype type: str
-    :param logbase: A positive number or e in string format to be used as base for log
-        based random sampling.
-    :type logbase: Union[float, str]
     :param rule: The specific type of random algorithm. Possible values include: "random",
         "sobol".
     :type rule: str
-    :param seed: An integer to use as the seed for random number generation.
+    :param seed: The seed for random number generation.
     :type seed: int
+    :param logbase: A positive number or e in string format to be used as base for log
+        based random sampling.
```

Comment:
Nit: quotes around "e" for clarity, and added "the":
```suggestion
    :param logbase: A positive number or "e" in string format to be used as the base for
        log-based random sampling.
```